### DISCUSSION TITLE
Notre serveur 🇸🇳

### DISCUSSION THREAD

**Discussion by dofbi** [Code-for-Senegal/joinDjolof#7] [General]
Quiconque possède un serveur et suffisamment de compétences pour le gérer peut créer sa propre Instance. Le logiciel qui gère un serveur pour Mastodon est entièrement libre et ouvert, mais l’hébergement d’un serveur, l’achat d’un nom de domaine, représente un certain coût, maintenir, modérer une Instance représente aussi du travail.

Selon la configuration technique et le nombre d’utilisateurs de l’Instance, son fonctionnement peut coûter entre quelques centaines de dollars par an et plusieurs milliers.

Mastodon est conçu pour être une plateforme sans publicité. Pour cette raison, Djolof à besoin financement participatif, mais aussi de travailleurs de volontaires.

**Comment by ByNatO** [comment]
Adding it : https://docs.joinmastodon.org/user/run-your-own/

**Reply by dofbi** [reply]
Voici une proposition d'architecture @ByNatO #8
